Control Plane stuff removed

djanderson opened the pull request:
This document should only cover the Data Plane. Removed 'Action Object' as it is not in the Data Plane. Moved 'Schedule Object' to Data Plane as it IS in the Data Plane.

bradeales committed:
Control Plane stuff removed

This document should only cover the Data Plane. Removed 'Action Object' as it is not in the Data Plane. Moved 'Schedule Object' to Data Plane as it IS in the Data Plane.

MichaelCotton reviewed (approved):
Yes, this is an improvement. Pls remove the following out-of-date text in Control plane/scos-sensor section: "(* note this is currently a private repository and will be released at a future date)"

bradeales committed:
Updated API link to reference auto-API

Removed old manually-generated API documentation. Link now points to auto-generated API documentation.

bradeales committed:
Added in link to SCOS Sensor repo.

MichaelCotton reviewed (approved):
Looks good
